Backpack/Common.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef BPK_Common_h
+#define BPK_Common_h
+
+#import <pthread.h>
+
+#define BPKAssertMainThread() NSAssert(defined(NS_BLOCK_ASSERTIONS) || 0 != pthread_main_np(), @"This method must be called on the main thread")
```

Comment:
Do we even need to check `NS_BLOCK_ASSERTIONS` ourselves? My impression is that `NSAssert` already does that

Reply:
Good point